.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Does this conform to the definition of the IDE config we have for SmallRye? ;-)

Reply:
nope, but the maven build does the job :-D

Reply:
consistency!! ;-)